### PR TITLE
Add suggestions to kickstart error message in liveinst

### PR DESCRIPTION
**Pull request by VladimirSlavik**
Resolves: [rhbz#1893428](https://bugzilla.redhat.com/show_bug.cgi?id=1893428)

I'm not sure if the text can reference Fedora editions, please discuss.

### CODE REVIEW

**Review by jkonecny12 — APPROVED** [review]
LGTM!

**Comment by VladimirSlavik** [comment]
@Gel0bmstu @mikhailnov What do you non-Fedora-distro people think about this?

**Comment by Gel0bmstu** [comment]
Hi, in ROSA Linux we use shell wrapper over _/usr/sbin/anaconda_ script, which allows user to perform kickstart install, even if hi load with livecd:

```
# --liveinst arg broke kickstart installation, so we should
# check if inst.ks setted in cmdline before begin installation.
# It's don't broke installation from GUI, because desktop icon
# run 'liveinst' script. 
LIVE_ARGS=""
if ! grep -q 'inst.ks' /proc/cmdline ; then
    LIVE_ARGS="--liveinst"
    if [ -n "$LIVE_BLOCK" ]; then
        LIVE_ARGS="$LIVE_ARGS --method=livecd:${LIVE_BLOCK}"
        # where LIVE_BLOCK is a path to /dev/mapper/live-base "file"
    fi
fi

....

exec anaconda ${LIVE_ARGS} "$@"
```

I think it is better to let the user install the system, than to confuse him with an incomprehensible message like "Kickstart is not supported on live install". However, the solution offered by @VladimirSlavik LGTM too!

**Comment by VladimirSlavik** [comment]
Thank you. I was concerned mainly because the script could run anywhere, but with this change it would mentions things specific to Fedora.

**Review by poncovka — CHANGES_REQUESTED** [review]
I have forgot to submit my review.

File: data/liveinst/liveinst [hunk 1]
```diff
@@ -110,7 +110,7 @@ for opt in `cat /proc/cmdline` $*; do
         ;;
     ks=*|kickstart=*|--ks=*|--kickstart=*|inst.ks=*|--inst.ks=*|inst.kickstart=*|--inst-kickstart=*)
         title="Configuration not supported"
-        text="Kickstart is not supported on live installs.  This installation will continue interactively."
+        text="Kickstart is not supported on live installs.  This installation will continue interactively.  Use kickstart with Server, IoT, or other non-Live images."
```
[the remark this answers is not in the document]
Comment (VladimirSlavik) [reply data/liveinst/liveinst]: Thank you, that's better.

File: docs/common-bugs.rst [hunk 2]
```diff
@@ -344,7 +344,7 @@ Automatic installation in Live OS
 
 :Issue: The reporter would like to run a kickstart installation in Live OS.
 :Solution: Kickstart installations in Live OS are not supported. Please, run the installation with
-    ``boot.iso``.
+    ``boot.iso`` ("netinst") for the Server edition of Fedora.
```
Comment (poncovka) [inline docs/common-bugs.rst:347]: I would try to expand the sentence a little more with something like:
`... or use a netinstall or standard ISO (for example, with the Server edition of Fedora).`

File: docs/common-bugs.rst [hunk 3]
```diff
@@ -344,7 +344,7 @@ Automatic installation in Live OS
 
 :Issue: The reporter would like to run a kickstart installation in Live OS.
```
Comment (poncovka) [inline docs/common-bugs.rst:345]: The reporter mentioned that it was difficult to find a solution for the message: `Kickstart is not supported on live installs. This installation will continue interactively.` Could you maybe document this message in the issue description?

**Comment by VladimirSlavik** [comment]
Oops, sorry, I did not think at all when I changed this. That's hardly better.

File: docs/common-bugs.rst (hunk 2 above)
Comment (VladimirSlavik) [reply docs/common-bugs.rst:347]: Is netinstall something else than boot.iso?

File: docs/common-bugs.rst (hunk 3 above)
Comment (VladimirSlavik) [reply docs/common-bugs.rst:345]: Yes, but I'll use the new one.

**Review by poncovka — APPROVED** [review]
Looks good to me. Thanks!

File: docs/common-bugs.rst (hunk 3 above)
Comment (VladimirSlavik) [reply docs/common-bugs.rst]: Used both - f33 is still "active", and f34 will be, so the old message will appear more often for at least a year.

File: docs/common-bugs.rst (hunk 2 above)
Comment (VladimirSlavik) [reply docs/common-bugs.rst]: Resolved out of band: netinstall is a subset of boot.iso. Being a boot.iso means the media is built to boot into anaconda. netinstall then has only that, while the other (unnamed) possibility is to have also packages on the media.

**Comment by mikhailnov** [comment]
We in ROSA install OS using rsync via Anaconda, so _any_ ISO is a LiveCD, either booted to multi-user.target or to anaconda.target. That is why we make workarounds described by @Gel0bmstu in some places.

**Pull request merged**